client/app/shared/pagination/pagination.html

```diff
@@ -1,4 +1,12 @@
 <div class="pagination-footer">
+  <span class="pull-left checkbox span-right-border" style="margin-top: 2px; margin-left: 18px" ng-if="vm.checkAll">
```

Comment:
@himdel any chance we can kill the inline styles for a class (`.checkallMarginOverride`? ) in _pagination.sass ?

Reply:
Yeah, makes sense :)

Reply:
Should be ready :)